src/tests/gfx_feature.edn

```diff
@@ -0,0 +1,67 @@
+(def timestep (/ 1.0 120.0))
+(defmesh root)
+(def BlendAdd {:Operation BlendOperation.Add :Src BlendFactor.One :Dst BlendFactor.One})
+(def BlendAlpha {:Operation BlendOperation.Add :Src BlendFactor.SrcAlpha :Dst BlendFactor.OneMinusSrcAlpha})
+(def BlendOne {:Operation BlendOperation.Add :Src BlendFactor.One :Dst BlendFactor.One})
+
+(defn spin-transform [t location]
+  (->
+   t >= .tmp-0
+   .tmp-0 (Math.Multiply 0.2) (Math.AxisAngleX) (Math.Rotation) >= .rotX
+   .tmp-0 (Math.Multiply 0.7) (Math.AxisAngleY) (Math.Rotation) >= .rotY
+   .tmp-0 (Math.Multiply 0.9) (Math.AxisAngleZ) (Math.Rotation) >= .rotZ
```

Comment:
Temporary variable can be skipped using [`(Sub)`](https://docs.fragcolor.xyz/shards/General/Sub/) (or its alias `(|)`.

```suggestion
   t
   (| (Math.Multiply 0.2) (Math.AxisAngleX) (Math.Rotation) >= .rotX)
   (| (Math.Multiply 0.7) (Math.AxisAngleY) (Math.Rotation) >= .rotY)
   (| (Math.Multiply 0.9) (Math.AxisAngleZ) (Math.Rotation) >= .rotZ)
```

Reply:
Forgot about that one, thanks!